### PR TITLE
manual: purchase: add a section on shipping timelines

### PR DESCRIPTION
**Pull request by whitequark**

### CODE REVIEW

File: docs/manual/src/purchase.rst [hunk 1]
```diff
@@ -16,3 +16,18 @@ Pre-assembled Glasgow devices and cases are available for purchase from the stor
...
+
+Shipping timelines
+------------------
+
+The CrowdSupply campaign was fully funded on February 2nd, 2021. The Early Bird devices were shipped to Mouser on September 30th, 2023. Understandably, many people have questions about their past or planned purchase, including:
+
+- When will my non Early Bird unit ship?
+- If I purchase a unit now, when will it ship?
+
+The campaign was deeply impacted by additional manufacturing challenges such as the COVID-19 pandemic, semiconductor and passive component shortages on top of the more usual and well-known ones. The answers to both of these questions is: **It ships when it ships**.
```
Comment (esden) [inline docs/manual/src/purchase.rst:31]: I would add the following to this line:

> We do work hard to get more hardware manufactured and shipped as soon as possible. If you want more recent updates than the CrowdSupply published updates, keep an eye on our chat channels. We have a weekly developer meeting on saturdays where @esden provides CrowdSupply campaign progress updates.

Comment (whitequark) [reply docs/manual/src/purchase.rst:31]: What should "@esden" link to?

Comment (esden) [reply docs/manual/src/purchase.rst:31]: Good question... maybe my mastodon? @esden@chaos.social as I do post things about progress there too, which is definitely more granular than CS updates.

Comment (esden) [reply docs/manual/src/purchase.rst:31]: That is still not a proper link. Sorry.. probably use this link: https://chaos.social/@esden

Comment (whitequark) [reply docs/manual/src/purchase.rst:31]: Done

Comment (esden) [reply docs/manual/src/purchase.rst:31]: Great thank you. It looks good to me. :)

**Pull request merged**